src/web/components/ClientSideTokenGeneration/CstgDomainsTable.scss

```diff
@@ -0,0 +1,59 @@
+.cstg-domains-management {
+  .cstg-domains-table {
+    width: 100%;
+    th:first-of-type,
+    td:first-of-type {
+      padding-left: 10px;
+      width: 21px;
+    }
+    th.action {
+      text-align: end;
+      padding-left: 140px;
+    }
+    tr {
+      .tristate-checkbox {
+        margin-right: 16px;
+      }
+
+      button[role='checkbox'] {
+        width: 21px;
+        height: 21px;
+      }
+      .input-container {
+        height: 26px;
+      }
+    }
+
+    .domain {
+      min-width: 220px;
+    }
+
+    .action-cell {
+      font-size: 0.8125rem;
+      display: flex;
+      justify-content: flex-end;
+      flex-wrap: wrap;
+
+      button {
+        padding: 5px 0 5px 10px;
+      }
+    }
+  }
+
+  .cstg-domains-management-actions {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+    padding-left: 11px;
+    margin-bottom: 10px;
+  }
+
+  .cstg-domains-management-delete-button {
+    color: var(--theme-radio-background);
```

Comment:
radio-background doesn't sound right for a delete button - is there a more appropriate theme value for this?

Reply:
I've add new theme value for it